schema.graphql

```diff
@@ -309,6 +309,7 @@ enum EventIdEnum {
   RegisterAssetMetadataGlobalType
   MetadataValueDeleted
   LocalMetadataKeyDeleted
+  AssetMediatorsAdded
   SetAssetMediators
```

Comment:
`SetAssetMediators` was on the release notes, but its not actually an event. Its `AssetMediatorsAdded` instead.

Postgres doesn't allow for removal of enums. The entire type needs to be dropped and then recreated.

If resync is fast now maybe its not a big deal to force one to clean up